content/advisories/2021-11-11-transaction-broadcast.zh-tw.md

```diff
@@ -0,0 +1,28 @@
+
++++
+title = "公告：對交易自動重新進行的回應"
+date = "2021-11-11"
+[ author ]
+  name = "Dogecoin Foundation"
++++
+
+幣安今日暫停了狗狗幣的提款功能，並指出他們發現狗狗幣的＂小問題＂。對此我們想做出回應：
```

Comment:
"The Foundation wishes to address this:"

"We" is replaced to "The Foundation", please correct it.